projects/plugins/social/src/class-meta-tags.php

```diff
@@ -133,14 +134,14 @@ public function should_render_twitter_cards_tags() {
 	 * @param int $height  The minimum height of the image. Optional. Defaults to 200.
 	 * @return array The featured image and dimensions. Empty array if no image is found.
 	 */
-	public function get_featured_image( $post_id = 0, $width = 200, $height = 200 ) {
+	public function get_featured_image( $post_id = null, $width = 200, $height = 200 ) {
 		$post = get_post( $post_id );
 
 		if ( ! has_post_thumbnail( $post ) ) {
 			return array();
 		}
```

Comment:
I just realised we might want to add an extra check here to see if `$post` is an actual post. Right now, you could pass in a non-existent post ID, and `$post` would return `null,` which would then make `has_post_thumbnail` check the global post object, rather than the post ID that was passed in.